static/xref/script.js

```diff
@@ -95,7 +95,9 @@ function getFormData() {
 }
 
 async function onSubmit(event) {
-  event.preventDefault();
+  if (event && typeof event.preventDefault === "function") {
```

Comment:
any particular reason for this check?

Reply:
wanted to reuse `onSubmit` below

Reply:
...to call `onSubmit` without any event

Reply:
ah, maybe put a comment that this is not a support check.

Reply:
makes sense